<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ldu-home-1" failures="1" tests="1" name="com.d3security.testcase.PhysicSystemTest" time="55.954" errors="0" timestamp="2023-07-09T15:38:36 PDT" skipped="0">
  <testcase classname="com.d3security.testcase.PhysicSystemTest" name="testPhysicAddIR" time="55.954">
    <failure message="[Step #9] [&lt;b&gt;&lt;font color=&quot;red&quot;&gt;Failed&lt;/font&gt;&lt;/b&gt;]: java.lang.AssertionError: IncidentReport created successfully expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: [Step #9] [<b><font color="red">Failed</font></b>]: java.lang.AssertionError: IncidentReport created successfully expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at com.d3security.util.StepInfo.failTest(StepInfo.java:123)
at com.d3security.util.Verify.verifyTrue(Verify.java:219)
at com.d3security.testcase.PhysicSystemTest.testPhysicAddIR(PhysicSystemTest.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
    <system-out>
      <![CDATA[[Step #1] [Message]:<b> Open Website URL:https://v2019.d3securityonline.net/V127Physical/VSOC/</b>
[Step #2] [Message]:<b> Sign In</b>
[Step #3] [Message]:<b> Dashboard is displaying</b>
[Step #4] [Message]:<b> Click Add New Icon</b>
[Step #5] [Message]:<b> Select dropdown Incident Report</b>
[Step #6] [Message]:<b> Create IR</b>
[Step #7] [Message]:<b> Add IncidentReport Window is opening</b>
[Step #8] [Message]:<b> Save</b>
<br><div style="text-align:left"><a href="F:\lucyWorkSpace\d3securityautomation\target\surefire-reports\html\202307915027FXApfG.jpg"><img src="F:\lucyWorkSpace\d3securityautomation\target\surefire-reports\html\202307915027FXApfG.jpg" alt="F:\lucyWorkSpace\d3securityautomation\target\surefire-reports\html\202307915027FXApfG" height="350"/></a></div>
<br>
[Step #9] [<b><font color="red">Failed</font></b>]: java.lang.AssertionError: IncidentReport created successfully expected [true] but found [false]]]>
    </system-out>
  </testcase> <!-- testPhysicAddIR -->
  <system-out>
    <![CDATA[[Step #1] [Message]:<b> Open Website URL:https://v2019.d3securityonline.net/V127Physical/VSOC/</b>
[Step #2] [Message]:<b> Sign In</b>
[Step #3] [Message]:<b> Dashboard is displaying</b>
[Step #4] [Message]:<b> Click Add New Icon</b>
[Step #5] [Message]:<b> Select dropdown Incident Report</b>
[Step #6] [Message]:<b> Create IR</b>
[Step #7] [Message]:<b> Add IncidentReport Window is opening</b>
[Step #8] [Message]:<b> Save</b>
<br><div style="text-align:left"><a href="F:\lucyWorkSpace\d3securityautomation\target\surefire-reports\html\202307915027FXApfG.jpg"><img src="F:\lucyWorkSpace\d3securityautomation\target\surefire-reports\html\202307915027FXApfG.jpg" alt="F:\lucyWorkSpace\d3securityautomation\target\surefire-reports\html\202307915027FXApfG" height="350"/></a></div>
<br>
[Step #9] [<b><font color="red">Failed</font></b>]: java.lang.AssertionError: IncidentReport created successfully expected [true] but found [false]]]>
  </system-out>
</testsuite> <!-- com.d3security.testcase.PhysicSystemTest -->
